packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -194,6 +197,31 @@ async function startWatcher(opts: SetupOpts) {
 
   await hotReloader.start()
 
+  // have to write this after starting hot-reloader since that
+  // cleans the dist dir
+  const routesManifestPath = path.join(distDir, ROUTES_MANIFEST)
+  const routesManifest: Partial<RoutesManifest> = {
+    version: 3,
+    caseSensitive: !!nextConfig.experimental.caseSensitiveRoutes,
+    basePath: nextConfig.basePath,
+    rewrites: opts.fsChecker.rewrites,
+    redirects: opts.fsChecker.redirects,
+    headers: opts.fsChecker.headers,
+    dataRoutes: [],
+    i18n: nextConfig.i18n || undefined,
+    skipMiddlewareUrlNormalize: nextConfig.skipMiddlewareUrlNormalize,
+  }
```

Comment:
Makes me a bit nervous that we're writing a partial manifest here. If the hot reloader only expects a portion of the routes manifest, perhaps it should be a specifically typed version of the manifest so that we can ensure that the right parameters are included in it. Today if a new parameter is used it's not going to warn due to the missing variable here.

Reply:
Makes sense can break this out to it's own type